apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/UrlSigner.PostPolicy.cs

```diff
@@ -0,0 +1,315 @@
+﻿// Copyright 2020 Google LLC
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     https://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Net;
+
+namespace Google.Cloud.Storage.V1
+{
+    public sealed partial class UrlSigner
+    {
+        /// <summary>
+        /// A known element that can be added to a <see cref="PostPolicy"/>.
+        /// </summary>
+        public enum PostPolicyKnownElement
```

Comment:
Possibly "standard" instead of "known"? (Standard vs custom is a fairly common distinction, I think.)